@utahdts/utah-design-system/react/enums/skeletonStyles.js

```diff
@@ -0,0 +1,8 @@
+/** @typedef {import('@utahdts/utah-design-system').SkeletonStyles} SkeletonStyles */
+
+/** @enum {SkeletonStyles} */
+export const skeletonStyles = {
+  CIRCULAR: /** @type {SkeletonStyles} */ ('skeleton--circle'),
+  LINEAR: /** @type {SkeletonStyles} */ ('skeleton--line'),
+  RECTANGULAR: /** @type {SkeletonStyles} */ ('skeleton--block'),
```

Comment:
Why do the constant names differ from the actual values? Seems like a rectangle should be called rectangle instead of block or vice versa a block should be called a block? Or maybe CSS world expects block/line and react world expects rectangular/linear?